install.md

```diff
@@ -22,7 +22,7 @@ To install Application Service Adapter:
     kubectl create ns tas-adapter-install
     ```
 
-1. Create an image pull secret to store your Tanzu Network credentials. These are required so that the cluster can pull images from the Tanzu Network registry.
+2. Create an image pull secret to store your Tanzu Network credentials. These are required so that the cluster can pull images from the Tanzu Network registry.
```

Comment:
@emalm if I remember correctly does the Docs tooling want these all to be `1.` and it owns the translation into the numbered list?

Reply:
reverted the changes